### PR TITLE
Fix documentation links

### PR DESCRIPTION
**Pull request by JihwanEom**
### Summary
- Fix broken documentation links

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by wonjuleee — DISMISSED** [review]

**Comment by vinnamkim** [comment]
- Fix for https://github.com/openvinotoolkit/datumaro/issues/971

File: README.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ CVAT annotations                             ---> Publication, statistics etc.
 ```
 <!--lint enable fenced-code-flag-->
 
-- [Getting started](https://openvinotoolkit.github.io/datumaro/docs/getting_started)
+- [Getting started](https://openvinotoolkit.github.io/datumaro/latest/docs/get-started/quick-start-guide)
 - [Features](#features)
-- [User manual](https://openvinotoolkit.github.io/datumaro/docs/user-manual)
-- [Developer manual](https://openvinotoolkit.github.io/datumaro/api)
```
Comment (vinnamkim) [inline README.md:21]: Would you add "Level Up" sections?

Comment (JihwanEom) [reply README.md:21]: Sure, reflected at https://github.com/JihwanEom/datumaro/commit/1a691ddc369e453d47c5d6009ea499ff757a8745

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Review by wonjuleee — APPROVED** [review]

**Pull request merged**